Sources/VitalHealthKit/HealthKit/Abstractions.swift

```diff
@@ -145,6 +145,49 @@ extension VitalHealthKitStore {
     case HKSampleType.quantityType(forIdentifier: .respiratoryRate)!:
       return [.vitals(.respiratoryRate)]
 
+    case
+      HKQuantityType.quantityType(forIdentifier: .dietaryBiotin)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryEnergyConsumed)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryCarbohydrates)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryFiber)!,
+      HKQuantityType.quantityType(forIdentifier: .dietarySugar)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryFatTotal)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryFatMonounsaturated)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryFatPolyunsaturated)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryFatSaturated)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryCholesterol)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryProtein)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryVitaminA)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryThiamin)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryRiboflavin)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryNiacin)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryPantothenicAcid)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryVitaminB6)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryBiotin)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryVitaminB12)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryVitaminC)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryVitaminD)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryVitaminE)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryVitaminK)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryFolate)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryCalcium)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryChloride)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryIron)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryMagnesium)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryPhosphorus)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryPotassium)!,
+      HKQuantityType.quantityType(forIdentifier: .dietarySodium)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryZinc)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryChromium)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryCopper)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryIodine)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryManganese)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryMolybdenum)!,
+      HKQuantityType.quantityType(forIdentifier: .dietarySelenium)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryWater)!,
+      HKQuantityType.quantityType(forIdentifier: .dietaryCaffeine)!:
```

Comment:
We might want to exclude these for now, as they are being synced individually.

Reply:
That or we stop syncing these as individual resources, and remap them to `.nutrition(.caffeine)` and `.nutrition(.water)` to `.meal`.

Reply:
Well there is not much harm in polling them for meal as well, right?

Reply:
Keeping the individual .water and .meal can be useful for apps that only track hydration for example.

And then polling them separately with intention to wrap them in a meal object on top of that.

Reply:
Ehm… I need to double check if there is harm. We used to enforce 1 HKSampleType -> 1 VitalResource for change observation purposes. But I might have changed that recently when breaking out timeseries from `VitalResource.activity`.

Reply:
Ah I see, let me know

Reply:
You will need to remove these two here. We already have dedicated switch cases for water & caffeine above. Add `.meal` in those two places  instead.

Reply:
```diff
      case HKSampleType.quantityType(forIdentifier: .dietaryWater)!:
-       return [.nutrition(.water)]
+       return [.meal, .nutrition(.water)]
      case HKSampleType.quantityType(forIdentifier: .dietaryCaffeine)!:
-       return [.nutrition(.caffeine)]
+       return [.meal, .nutrition(.caffeine)]
```

Reply:
Thanks. Though I just realised that we don't even have place for caffeine in our Meal model.
Should we update it?
(and it looks like none of the 3 apps I have on ios update the water nutrient)

Reply:
wdyt?

Reply:
Honestly I would just exclude these two from meal syncing. Caffeine is rarely part of food nutritional labels. Water is tracked separately.